src/apps/replication/client_ddl_tool/main.cpp

```diff
@@ -0,0 +1,102 @@
+#include <dsn/service_api_c.h>
+#include <dsn/ports.h>
+#include <dsn/dist/replication/empty_app.h>
+#include <dsn/dist/replication/client_ddl.h>
+#include <iostream>
+
+using namespace dsn::client;
+
+void usage(char* exe)
+{
+    std::cout << "Usage:" << std::endl;
+    std::cout << "\t" << exe << " <config.ini> create_app -name <app_name> -type <app_type> [-pc partition_count] [-rc replication_count]" << std::endl;
+    std::cout << "\t" << exe << " <config.ini> drop_app -name <app_name>" << std::endl;
```

Comment:
add two api
list_apps to list all apps
list_app to display info for a specified app.

both APIs can print info to screen or file.